examples/starter-policy/fastapi/README.md

```diff
@@ -0,0 +1,235 @@
+
+The `Organization` resource has two roles:
+- the `viewer` role (which grants `view`), and
+- the `owner` role (which grants both `view` and `edit`)
+
+By assigning roles to an Actor as a fact, we can authorize access. Roles are assigned to individual instances of a resource - in this case, we'll consider `Organization{"acme"}`.
+
+### `GET /org/acme`
+
+The `get_organization` route is enforced:
+
+```python
+@app.get("/org/{id}")
+@oso.enforce("{id}", "view", "Organization")
+```
+
+
+To access this route, you'll need to provide the `view` permission. This can be accomplished by adding a fact, which gives `User{"anonymous"}` the `viewer` role:
+- `has_role(User{"anonymous"}, "viewer", Organization{"acme"})`
+
+Now, the following request should succeed:
+
+```bash
+curl localhost:8000/org/acme
+```
+
+### `POST /org/acme`
+
+The `post_organization` route is enforced:
+
+``` python
+@app.post("/org/{id}")
+@oso.enforce("{id}", "edit", "Organization")
+```
+
+This time, instead of the `view` permission, you'll need to provide the `edit` permission.
+
+This can be accomplished by adding a fact which gives `User{"anonymous"}` the `owner` role:
+- `has_role(User{"anonymous"}, "owner", Organization{"acme"})`
+
+Now, the following request should succeed:
+
+```bash
+curl -X POST localhost:8000/org/acme
+```
+
+(This route doesn't require a request body, it is just looking for a POST request.)
+
+## Constructing a Relation with Facts
+
+The resource block definition for `Repository` contains a relation, `repository_tenant`:
+
+```
+resource Repository {
+  # ...
+  relations = { repository_tenant: Organization };
+  # ...
+}
+```
+
+This allows **associating a Repository with an Organization**. We declare this with a fact, which references:
+- a specific instance of a `Repository`, and
+- a specific instance of an `Organization`.
+
+For example, if we wanted to declare a relationship from `Repository{"code"}` to `Organization{"acme"}`, we could add the following fact to Oso Cloud:
+
+- `has_relation(Repository{"code"}, "repository_tenant", Organization{"acme"})`
+
+This is relevant for a few permissions, defined on the `Repository` resource:
+```
+resource Repository {
+  # ...
+  "view" if "viewer" on "repository_tenant";
+  "edit" if "owner" on "repository_tenant";
+  # ...
+}
+```
+
+Assuming we declared the relationship between `Repository{"code"}` and `Organization{"acme"}`, this means:
+- if an actor has the `viewer` role on `Organization{"acme"}`, they will have the `view` permission on `Repository{"code"}`
+- if an actor has the `owner` role on `Organization{"acme"}`, they will have the `edit` permission on `Repository{"code"}`
+
+This gives us some flexbility - we can allow access with a few different approaches.
+
+## `GET /repo/code`
+
+The `get_repository` route is enforced:
+
+```python
+@app.get("/repo/{id}")
+@oso.enforce("{id}", "view", "Repository")
+```
+
+### Allowing `view` through `Repository{"code"}`
```

Comment:
I think we should remove this because the prior example already shows how you may assign permissions directly. If I understand correctly, the goal here is to showcase that it can also be done via relations. If I were someone new to Oso, this section would be confusing to me because there's a lot being shown.